src/mpyl/projects/versioning.py

```diff
@@ -161,6 +163,36 @@ def upgrade(self, previous_dict: ordereddict) -> ordereddict:
         return previous_dict
 
 
+class ProjectUpgraderFive(Upgrader):
+    target_version = 5
+
+    def upgrade(self, previous_dict: ordereddict) -> ordereddict:
+        # update the env var url's
+        deployments = previous_dict.get("deployments", [])
+        for deployment in deployments:
+            properties = deployment.get("properties")
+
+            if properties:
+                for env_var in properties.get("env", []):
+                    for key, value in env_var.items():
+                        regex = (
+                            r"http://([a-z]+)\.("
+                            + f"{NAMESPACE_PLACEHOLDER}"
+                            + r"|[a-z]+)\.svc"
```

Comment:
Similar issue as above, namespaces can have dashes: [`http://energyConsumers.vdb-services.svc`](https://github.com/Vandebron/argocd/blob/b2ffcee05d924e1610f0c48e00b3e7ce120040aa/acce/deployments/Vandebron/bpm-app/manifests/deployment-bpm-app.yaml#L49)

```suggestion
                            + r"|[a-z\-]+)\.svc"
```